src/core/psi_container.jl

```diff
@@ -290,6 +288,10 @@ function _encode_for_jump(name::Symbol)
     return name
 end
 
+function decode_symbol(name::Symbol)
```

Comment:
The encode and decode functions should be symmetric.  We should change `_encode_for_jump` to `encode_symbol`.  And maybe `_JUMP_NAME_DELIMITER` needs a better name.